crates/store/re_log_encoding/src/decoder/mod.rs

```diff
@@ -8,27 +8,15 @@ use std::io::Read;
 use re_build_info::CrateVersion;
 use re_log_types::LogMsg;
 
+use crate::codec;
 use crate::FileHeader;
 use crate::MessageHeader;
+use crate::VersionPolicy;
 use crate::OLD_RRD_HEADERS;
 use crate::{Compression, EncodingOptions, Serializer};
 
 // ----------------------------------------------------------------------------
 
-/// How to handle version mismatches during decoding.
-#[derive(Copy, Clone, Debug, PartialEq, Eq)]
-pub enum VersionPolicy {
```

Comment:
(could we have done this move in a separate small prep PR just to make this one slightly smaller and focused? )

Reply:
Yes, and it's still possible. As I submitted it, I honestly didn't think the PR would be big enough for it to matter, but at the very least I could've polished up the commit history to make it easier to review. I'll keep that in mind for future PRs, or if you want I can split up this one as well

Reply:
Actually, I will just split it up and untangle the dependencies in a separate PR first.

Reply:
thanks, I think it will reduce the size of this one and make it easier to review